src/css/main.scss

```diff
@@ -58,11 +58,16 @@
   --sn-stylekit-general-border-radius: 2px;
 
   --sn-stylekit-simplified-chinese-font: "Microsoft Yahei", "微软雅黑体";
-  // consolas and monaco don't look good on desktop
-  --sn-stylekit-monospace-font: "Ubuntu Mono", courier, monospace;
+  --sn-stylekit-monospace-font:
+    "SFMono-Regular", Consolas, "Liberation Mono",
+    Menlo, "Ubuntu Mono", "Courier New", monospace;
   --sn-stylekit-sans-serif-font: -apple-system, BlinkMacSystemFont, "Segoe UI",
     "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
     "Helvetica Neue", var(--sn-stylekit-simplified-chinese-font), sans-serif;
+  --sn-stylekit-editor-font-family: var(--sn-stylekit-sans-serif-font);
+
+  --sn-stylekit-theme-type: "light";
+  --sn-stylekit-theme-name: "sn-light";
```

Comment:
I think it may be better to not use quotations here because the consumer will have to strip them:
```css
  --sn-stylekit-theme-type: light;
  --sn-stylekit-theme-name: sn-light;
```

For example, if we read the CSS variables using JS like this:
```js
  const whiteSpaceRegExp = /\s+/g;
  const themeType = getComputedStyle(document.documentElement)
    .getPropertyValue('--sn-stylekit-theme-type')
    .replace(whiteSpaceRegExp, '');
  console.log('theme type', '1' + themeType + '2');
```
Then we'll get:
```
theme type 1"light"2
```
So if we're testing if it's a light theme, we won't be able to use this:
```js
  if (themeType === 'light') {
    return true;
  }
```
If we include quotes in the variable definition, then we would have to include quotes in the JS, which is messier and more susceptible to error:
```js
  if (themeType === '"light"') {
    return true;
  }
```
Are there any benefits of using quotations that outweigh these concerns?

Reply:
> Are there any benefits of using quotations that outweigh these concerns?

I don't think there is, no! I've updated the PR